pkg/controller/nodelifecycle/metrics.go

```diff
@@ -79,6 +79,25 @@ var (
 		},
 		[]string{"zone"},
 	)
+
+	updateNodeHealthDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      nodeControllerSubsystem,
+			Name:           "update_node_health_duration_seconds",
```

Comment:
Metrics names should be defined as const similar to other ones. i.e. `evictionsTotalKey`